practice-two/src/js/script.js

```diff
@@ -166,6 +166,47 @@ const saveData = () => {
     });
   }
   localStorage.setItem(UsersKey, JSON.stringify(existingEntries));
+  renderUserTable();
+}
+
+/**
+ * Show user data in table
+ */
+const renderUserTable = () => {
+  // Parse any JSON previously stored in allEntries
+  let existingEntries = JSON.parse(localStorage.getItem(UsersKey));
+
+   // If existingEntries is null, array will be created
+   if(existingEntries === null) {
+    existingEntries = [];
+  } else {
+    let tableInstead = `
+    <tbody> 
+      <tr>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+        <td></td>
+      </tr>
+    </tbody>`;
+  
+    existingEntries.map((value) => {
```

Comment:
Do not use map, because map return an array. So map function is used for converting an existing array to new array.

Use forEach, for instead